pkg/sensors/tracing/kprobe_test.go

```diff
@@ -5783,7 +5783,7 @@ spec:
 
 func TestKprobeListSyscallDupsRange(t *testing.T) {
 	if !kernels.MinKernelVersion("5.3.0") {
-		t.Skip("TestCopyFd requires at least 5.3.0 version")
+		t.Skip("TestKprobeListSyscallDupsRange requires at least 5.3.0 version")
```

Comment:
Can you please split these changes into another patch?

Reply:
I was just fixing a copy and paste error. I didn't think it would be an issue, but I can remove these fixes.

Reply:
Changes moved to a separate commit.